src/python/foglamp/rest_service/routes.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from foglamp.rest_service import api
+
+
+def setup(app):
+    # app.router.add_route('POST', '/foglamp/a-post-req', api.a_post_req, expect_handler = aiohttp.web.Request.json)
+    app.router.add_route('GET', '/foglamp/ping', api.ping)
+
+    # enable cors only in dev mode ?!
+    enable_cors(app)
+    # a live debugger (watcher) for requests
+    # this will neutralize error middleware
+    enable_debugger(app)
+
+
+def enable_cors(app):
```

Comment:
This allows us to access the api from the requests which are not from the same origin. https://en.wikipedia.org/wiki/Same-origin_policy 
For frontend, Once we agree that nginx is good to go for serving static files and as a reverse proxy to api calls. This can be made disabled (but we may may want to keep it in dev mode, as is.) For the api to api communication, we may want to keep this enabled in production mode too, but for specific ports and IP (not *).